src/index.py

```diff
@@ -1,5 +1,46 @@
+
+# Import of the modules
+
 import yfinance as yf
 import plotly.graph_objs as go
+import plotly.io as pio
+import time
+from selenium import webdriver
+import os
+from random import randint
+from asciimatics.screen import Screen
```

Comment:
please remove any code lines which include asciimatics. they are not required for now